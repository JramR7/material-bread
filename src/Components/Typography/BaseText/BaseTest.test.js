import React from 'react';
import BaseText from './BaseText';

import renderer from 'react-test-renderer';

const props = {
  theme: {
    text: {},
    textColor: {
      primary: 'red',
      secondary: 'blue',
    },
  },
};

describe('BaseText', () => {
  const tree = renderer.create(<BaseText {...props} />);

  test('renders', () => {
    expect(tree.toJSON()).toMatchSnapshot();
  });

  test('renders with different theme color', () => {
    tree.update(<BaseText {...props} themeColor="secondary" />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
