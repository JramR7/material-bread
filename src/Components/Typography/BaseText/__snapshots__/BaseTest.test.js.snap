// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseText renders 1`] = `
<Text
  style={
    Array [
      Object {},
      undefined,
      Object {
        "color": "red",
        "marginBottom": 0,
        "textAlign": "left",
      },
      undefined,
    ]
  }
/>
`;

exports[`BaseText renders with different theme color 1`] = `
<Text
  style={
    Array [
      Object {},
      undefined,
      Object {
        "color": "blue",
        "marginBottom": 0,
        "textAlign": "left",
      },
      undefined,
    ]
  }
/>
`;
